packages/block-editor/src/components/border-radius-control/utils.js

```diff
@@ -117,3 +117,106 @@ export function hasDefinedValues( values ) {
 
 	return !! filteredValues.length;
 }
+
```

Comment:
Most of these are copy/pastes (kind of) from the spacing sizes one.

Reply:
If we don't end up extracting common utils, there are a few references to spacing presets here that could be cleaned up.